crates/runtime/src/accelerated_table.rs

```diff
@@ -68,69 +68,69 @@ mod synchronized_table;
 
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Unable to get data from connector: {source}"))]
+    #[snafu(display("Failed to get data from the connector.\n{source}"))]
     UnableToGetDataFromConnector { source: DataFusionError },
 
-    #[snafu(display("Dataset refresh failed with error: {source}"))]
+    #[snafu(display("Failed to refresh the dataset.\n{source}"))]
     FailedToRefreshDataset { source: DataFusionError },
 
-    #[snafu(display("Unable to scan table provider: {source}"))]
+    #[snafu(display("Failed to read from the dataset.\n{source}"))]
```

Comment:
This feels like it could have the same error message as above? `Failed to get data from the connector`